src/inbox/mod_inbox_rdbms.erl

```diff
@@ -135,18 +138,25 @@ remove_domain(HostType, LServer) ->
 
 %% This function was not refatorected to use the generic upsert helper
```

Comment:
Just an observation, maybe we could _refactorect_ this line as well as we are changing the code 🙂